ProcessLib/SmallDeformationWithLIE/Common/MeshUtils.cpp

```diff
@@ -8,12 +8,53 @@
 
 #include "MeshUtils.h"
 
+#include "MeshLib/MeshSearch/NodeSearch.h"
 
 namespace ProcessLib
 {
 namespace SmallDeformationWithLIE
 {
 
+namespace
+{
+
+class IsCrackTip
+{
+public:
+    explicit IsCrackTip(MeshLib::Mesh const& mesh)
+        : _fracture_element_dim(mesh.getDimension()-1)
+    {
+        _is_internal_node.resize(mesh.getNumberOfNodes(), true);
+
+        MeshLib::NodeSearch nodeSearch(mesh);
+        nodeSearch.searchBoundaryNodes();
+        for (auto i : nodeSearch.getSearchedNodeIDs())
+            _is_internal_node[i] = false;
```

Comment:
Is it only a short-cut for later computation to store whether a node is internal or is it truly essential to know that?